modular_zubbers/code/modules/antagonists/malf/doomsday.dm

```diff
@@ -0,0 +1,12 @@
+/obj/machinery/doomsday_device/trigger_doomsday()
+	var/list/bombable = SSmapping.levels_by_trait(ZTRAIT_STATION)
+	for(var/obj/machinery/power/apc/power in world)
+		if(power.z == locate(power.z) in bombable)
+			explosion(power, light_impact_range = 15, explosion_cause = src, adminlog = TRUE)
+			sleep(1)
```

Comment:
probs shouldnt use sleeps as it locks shit the fuck up

Reply:
the mc also cant control it, easy fix is just stoplag(1) otherwise make it a timer with callbacks so you can moderate the amount it locks shit up.